src/sentry/static/sentry/app/views/acceptProjectTransfer.tsx

```diff
@@ -45,24 +58,21 @@ class AcceptProjectTransfer extends AsyncView {
   };
 
   renderError(error) {
-    let disableLog = false;
+    let disableLog: boolean = false;
```

Comment:
probably don't need the explicit `boolean` type declaration as it's implied by setting `disableLog` to `false`
```suggestion
    let disableLog = false;
```

Reply:
agree!